index.html

```diff
@@ -280,7 +280,9 @@ <h1 id="kittens">Kittens!</h1>
 
     <section class="sections">
       <h1 id="cool-stuff">Cool Stuff</h1>
-
+        <p>
+          <b><u>Content for cool stuff coming soon!</u></b>
+        </p>
```

Comment:
The `p` element should be aligned with the `h1` element because both are children of the `section` element